src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if (HAVE_GSSFW_HEADERS)
+   find_library(LIBGSS NAMES GSS)
+else()
+   find_library(LIBGSS NAMES gssapi_krb5)
+endif()
+
+if(LIBGSS STREQUAL LIBGSS-NOTFOUND)
+    message(FATAL_ERROR "Cannot find libgssapi_krb5 and System.Net.Security.Native cannot build without it. Try installing libkrb5-dev (or the appropriate package for your platform)")
```

Comment:
> Which other library are you referring to?

We're looking for two different sets of headers, but here we're only recommending one package.